modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -113,6 +108,14 @@ public int call(final String[] args, final Cfg cfg) {
                     }
                     System.out.println("p2p: " + cfg.getNet().getP2p().getIp() + ":" + cfg.getNet().getP2p().getPort());
                     break;
+                case "-s":
+                    File store = new File(CfgApiRpcSsl.SSL_KEYSTORE_DIR);
+                    if (store.mkdir()) {
+                        System.out.println("\nCreated sslKeystore directory.");
+                    } else {
+                        System.out.println("\nThe sslKeystore directory already exists.");
+                    }
+                    break;
```

Comment:
I think -s should generate the ssl file for the user also create the folder. not just create a folder.
Which means we might warp the openssl tool into binary release.